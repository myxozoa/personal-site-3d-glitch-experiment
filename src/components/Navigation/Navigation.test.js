import React from 'react';
import { shallow } from 'enzyme';

import Component from './index';

describe('<Navigation />', () => {
  it('renders correctly', () => {
    const component = shallow(<Component />);

    expect(component).toMatchSnapshot();
  });
});