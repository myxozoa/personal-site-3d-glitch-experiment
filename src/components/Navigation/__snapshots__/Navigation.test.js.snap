// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> renders correctly 1`] = `
<nav
  className="container"
>
  <ForwardRef
    className="navLink glitch"
    data-text="home"
    style={
      Object {
        "--offset": "1.5s",
      }
    }
    to="/"
  >
    home
  </ForwardRef>
  <ForwardRef
    className="navLink glitch"
    data-text="projects"
    style={
      Object {
        "--offset": "1.5s",
      }
    }
    to="/projects"
  >
    projects
  </ForwardRef>
  <ForwardRef
    className="navLink glitch"
    data-text="contact"
    style={
      Object {
        "--offset": "1.5s5",
      }
    }
    to="/contact"
  >
    contact
  </ForwardRef>
</nav>
`;
